designer/server/src/routes/file/file.js

```diff
@@ -41,16 +41,16 @@ export default [
             sessionNames.validationFailure.fileDownload
           )[0]
 
-          const email = await server.methods.state.get(
-            credentials.user.id,
-            sessionNames.fileDownloadPassword
-          )
-
-          if (email && !validation.formValues.email) {
-            validation.formValues.email = email
-          }
+          const email =
+            (await server.methods.state.get(
+              credentials.user.id,
+              sessionNames.fileDownloadPassword
+            )) ?? ''
```

Comment:
Did the last refactor revert this?

```suggestion
          const email = await server.methods.state.get(
            credentials.user.id,
            sessionNames.fileDownloadPassword
          )
```

Reply:
refactor done